src/chi/components/tables/tables.scss

```diff
@@ -134,6 +134,60 @@ th {
     }
   }
 
+  &.-portal {
+    thead {
+      tr {
+        th {
+          background-color: set-color(grey, 25);
+          border-bottom: 0;
+          font-weight: 700;
+          padding-bottom: 0.75rem;
+          padding-top: 0.75rem;
+
+          &:nth-child(2) {
```

Comment:
@janibolkvadze please remove this selector as it can be achieved with our alignment classes for tables.